### PR TITLE
Add contribution style guide

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Adds a style guide for content contributors

### CODE REVIEW

File: src/content/contributing/style-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+---
+title: Style Guide
+description: Style Guide for ethereum.org
+lang: en
+sidebar: true
+---
+
+# Ethereum.org Style Guide {#style-guide}
+
+Content on [Etheruem.org](http://etheruem.org) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:10]:
```suggestion
Content on [Etheruem.org](/) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
```

File: src/content/contributing/style-guide/index.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+## Who can submit content to Ethereum.org {#who-can-help}
+
+Anyone! [Ethereum.org](http://ethereum.org) is entirely open source, and many of its best pages are submitted by curious learners who expanded their notes into documentation pages now living on the site!
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:14]: Would either remove the redundant linking to the homepage and just make this text, or if we're going to keep the links, they should just link to `/` so it's not treated as an external link.

```suggestion
Anyone! Ethereum.org is entirely open source, and many of its best pages are submitted by curious learners who expanded their notes into documentation pages now living on the site!
```
Same thing with the others below

File: src/content/contributing/style-guide/index.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+#### Developers
+
+Developers or others who want technical information about Ethereum.
+
+#### Enterprises
+
+People, businesses, and other organizations who want to understand Ethereum's value in an enterprise setting
+
+Because [Ethereum.org](http://ethereum.org) caters to a wide variety of users, all content on the site should make an effort to explain any technologies and concepts as simply as possible.
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:32]: Would keep ethereum.org lowercased if not starting a sentence:
```suggestion
Because ethereum.org caters to a wide variety of users, all content on the site should make an effort to explain any technologies and concepts as simply as possible.
```
or 
```suggestion
Because [ethereum.org](/) caters to a wide variety of users, all content on the site should make an effort to explain any technologies and concepts as simply as possible.
```

File: src/content/contributing/style-guide/index.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+> e.g. _ "Ethereum will take over the world because..."_
+
+**Hostile or confrontational language aimed at any organization or person**
+
+> e.g. _ "Company X is bad because they are centralized!"_
+
+**Politically charged rhetoric**
+
+> e.g. _ "This political party are better for decentralization because..."_
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:48]:
```suggestion
> e.g. _ "This political party is better for decentralization because..."_
```

File: src/content/contributing/style-guide/index.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+### Use American English {#american-english}
+
+For words that have multiple spellings, use American English over British English.
+
+**For example:**
+
+- `decentralized` over `decentralised`
+- `color` over `colour`
+- `analyze` over `analyse`
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:64]: Is it official now? 😆  We've got some clean up to do

File: src/content/contributing/style-guide/index.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+### Acroynyms {#acroynyms}
+
+When introducing an unfamiliar acronym, spell out the full term, and put the acronym in parentheses. Put both the full term and acronym in bold.
+
+**For example:**
+
+"Ethereum, like Bitcoin, currently uses a consensus protocol called **Proof-of-work (PoW)**."
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:72]: Should we suggest here to link terms to the glossary if possible?

File: src/content/contributing/style-guide/index.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+
+### Consistency {#consistency}
+
+Many of the topics covered on [Ethereum.org](http://ethereum.org) are technically complex. To reduce confusion to the reader, terms should be used consistently. For example, don't cycle back-and-forth between proof-of-work and PoW at random.
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:76]: This goes to a point I made on #3653... should we encourage that the abbreviation be used after the term has been introduced on any given page?

File: src/content/contributing/style-guide/index.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+**Correct usage:**
+
+- Proof-of-stake
+- proof-of-stake
+
+**Incorrect usage:**
+
+- Proof-of-Stake
+- Proof of stake
+- proof of stake
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:122]: Again, just want to double-check this is accurate and proper. @ryancreatescopy You made comments about this in the past, do you recall where you found the proper english rules for these? (also commented about this in #3653)

File: src/content/contributing/style-guide/index.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+
+> _ "The man bought a car."_
+
+**Passive voice sentence:** a target acts on an actor
+
+> _ "The car was bought by a man."_
+
+[Read more on active voice](https://www.grammarly.com/blog/active-vs-passive-voice/)
+
+_This isn't an easy one, especially for non-native English speakers. If you aren't sure, don't worry. We'll help with this one._
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:138]: Perhaps broaden the offer for us to help?
```suggestion
_This isn't an easy one, especially for non-native English speakers. If you aren't sure, don't worry. We'll help with any of these._
```

**Review by wackerow — CHANGES_REQUESTED** [review]
@minimalsm Nice! This is great and should definitely help keep things cleaner and more consistent moving forward. Left some comments, let me know your thoughts =)

**Comment by wackerow** [comment]
@minimalsm One other thought I just had, we should include a section about standardizing capitalization surrounding ETH terms... like
"Ethereum" not "ethereum"
"ether" not "Ether"
"ETH" not "eth / Eth"

File: src/content/contributing/style-guide/index.md (hunk 5 above)
Comment (minimalsm) [reply src/content/contributing/style-guide/index.md:64]: Gotta pick one! I'd be more than happy to default to her Majesty's English on this occasion if we can agree on it 😂

File: src/content/contributing/style-guide/index.md [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+![How to mint your NFT](./mintYourNFT.gif)
+
+<--- Bad --->
+
+![How to mint your NFT](https://cdn-images-1.medium.com/max/2000/0342fj_fsdfs.gif)
+```
+
+### Using Emojis {#images}
+
+Everyone loves emojis <Emoji text="🥰" size={1} /> To stanardize the appearence of all Emojis across browsers [Ethereum.org](http://ethereum.org) uses an `Emoji` component.
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:172]:
```suggestion
Everyone loves emojis <Emoji text="🥰" size={1} /> To standardize the appearance of all Emojis across browsers [Ethereum.org](http://ethereum.org) uses an `Emoji` component.
```

File: src/content/contributing/style-guide/index.md [hunk 11]
```diff
@@ -0,0 +1,188 @@
...
+
+For words that have multiple spellings, use American English over British English.
+
+**For example:**
+
+- `decentralized` over `decentralised`
+- `color` over `colour`
+- `analyze` over `analyse`
+
+### Acroynyms {#acroynyms}
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:66]:
```suggestion
### Acronyms {#acronyms}
```

File: src/content/contributing/style-guide/index.md (hunk 4 above)
Comment (minimalsm) [reply src/content/contributing/style-guide/index.md:48]: @wackerow Haha that seems really weird to me. In the UK we use normally use 'are' for collective nouns. Another nuance to look out for.

File: src/content/contributing/style-guide/index.md (hunk 9 above)
Comment (minimalsm) [reply src/content/contributing/style-guide/index.md:138]: Good idea 👍

File: src/content/contributing/style-guide/index.md (hunk 10 above)
Comment (minimalsm) [reply src/content/contributing/style-guide/index.md:172]: Wish I could chalk this up to intl but I just can't spell it seems.

File: src/content/contributing/style-guide/index.md [hunk 12]
```diff
@@ -0,0 +1,188 @@
...
+- Ethereum Mainnet
+
+**Incorrect usage:**
+
+- main net
+- mainnet
+- Main net
+- Ethereum mainnet
+
+### Proof-of-work {#proof-of-work}
```
[the remark this answers is not in the document]
Comment (minimalsm) [reply src/content/contributing/style-guide/index.md]: Combined them

File: src/content/contributing/style-guide/index.md (hunk 7 above)
Comment (minimalsm) [reply src/content/contributing/style-guide/index.md:76]: Commented in #3653 but I lean slightly towards using the full term

File: src/content/contributing/style-guide/index.md (hunk 8 above)
Comment (minimalsm) [reply src/content/contributing/style-guide/index.md]: See Ryan's response in discord for context. After his input, I'm fine with keeping this as is if no one opposes it. 

But also happy to change it. My concern is that it becomes wordy and hard to follow.

**Comment by minimalsm** [comment]
@wackerow
- added Ethereum section
- added ETH/ether section
- combined PoW and PoS sections
- added all corrections suggested

File: src/content/contributing/style-guide/index.md [hunk 13]
```diff
@@ -0,0 +1,200 @@
...
+
+Content on [Etheruem.org](/) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
+
+## Who can submit content to Ethereum.org {#who-can-help}
+
+Anyone! Ethereum.org is entirely open source, and many of its best pages are submitted by curious learners who expanded their notes into documentation pages now living on the site!
+
+## Audience {#audience}
+
+[Ethereum.org](http://ethereum.org) content caters to a large and varied audience. Loosely we can categorize them as:
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:18]:
```suggestion
Ethereum.org content caters to a large and varied audience. Loosely we can categorize them as:
```

File: src/content/contributing/style-guide/index.md [hunk 14]
```diff
@@ -0,0 +1,200 @@
...
+
+When introducing an unfamiliar acronym, spell out the full term, and put the acronym in parentheses. Put both the full term and acronym in bold.
+
+**For example:**
+
+"Ethereum, like Bitcoin, currently uses a consensus protocol called **Proof-of-work (PoW)**."
+
+### Consistency {#consistency}
+
+Many of the topics covered on [Ethereum.org](http://ethereum.org) are technically complex. To reduce confusion to the reader, terms should be used consistently. For example, don't cycle back-and-forth between proof-of-work and PoW at random.
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:76]:
```suggestion
Many of the topics covered on ethereum.org are technically complex. To reduce confusion to the reader, terms should be used consistently. For example, don't cycle back-and-forth between proof-of-work and PoW at random.
```

File: src/content/contributing/style-guide/index.md [hunk 15]
```diff
@@ -0,0 +1,200 @@
...
+**Incorrect usage:**
+
+- main net
+- mainnet
+- Main net
+- Ethereum mainnet
+
+### Proof-of-work / Proof-of-stake {#proof-of}
+
+Proof-of-work should be capitalized at the beginning of a sentence or when the term is introduced for the first time on a page, in any other instance, all letters should be lower case. In either case, proof-of-work should be hyphenated between each word.
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:109]: > or when the term is introduced for the first time on a page

Should we be capitalizing this mid-sentence ever? I don't think capitalizing terms that are new to a page is a convention we've done elsewhere.

Comment (wackerow) [reply src/content/contributing/style-guide/index.md:109]: Also, perhaps we should just broaden this to proof-of-whatever.... show the PoW examples, but note that would similarly apply to proof-of-stake, proof-of-humanity, proof-of-individuality, proof-of-authority, etc.

File: src/content/contributing/style-guide/index.md [hunk 16]
```diff
@@ -0,0 +1,200 @@
...
+description: Style Guide for ethereum.org
+lang: en
+sidebar: true
+---
+
+# Ethereum.org Style Guide {#style-guide}
+
+Content on [Etheruem.org](/) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
+
+## Who can submit content to Ethereum.org {#who-can-help}
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:12]:
```suggestion
## Who can submit content to ethereum.org {#who-can-help}
```

File: src/content/contributing/style-guide/index.md [hunk 17]
```diff
@@ -0,0 +1,200 @@
+---
+title: Style Guide
+description: Style Guide for ethereum.org
+lang: en
+sidebar: true
+---
+
+# Ethereum.org Style Guide {#style-guide}
+
+Content on [Etheruem.org](/) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:10]: Another small convention we've been using throughout the site is not capitalizing the website name of ethereum.org (unless it starts a sentence).
```suggestion
Content on [etheruem.org](/) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
```

File: src/content/contributing/style-guide/index.md [hunk 18]
```diff
@@ -0,0 +1,200 @@
...
+The London Upgrade is live <Emoji text="❤️" size={1} />
+
+<--- Bad --->
+
+The London Upgrade is live ❤️
+```
+
+## Anything else? {#anything-else}
+
+Like everything on Ethereum.org, the style guide is an open-source work-in-progress with room for improvement. If there is anything you think should be added to improve this document please [edit it on GitHub](https://github.com/ethereum/ethereum-org-website/blob/dev/src/content/contributing/style-guide/index.md).
```
Comment (wackerow) [inline src/content/contributing/style-guide/index.md:200]:
```suggestion
Like everything on ethereum.org, the style guide is an open-source work-in-progress with room for improvement. If there is anything you think should be added to improve this document please [edit it on GitHub](https://github.com/ethereum/ethereum-org-website/blob/dev/src/content/contributing/style-guide/index.md).
```

**Review by wackerow — CHANGES_REQUESTED** [review]
Looking solid, just left a couple last comments. Looks great overall!

File: src/content/contributing/style-guide/index.md [hunk 19]
```diff
@@ -0,0 +1,200 @@
+---
```
Comment (samajammin) [inline src/content/contributing/style-guide/index.md:1]: We should link to this page from https://ethereum.org/en/contributing/, probably the "How to work on ethereum.org" section.

File: src/content/contributing/style-guide/index.md (hunk 17 above)
Comment (samajammin) [inline src/content/contributing/style-guide/index.md:10]: Somewhere (perhaps the intro section) I think we should link to the design principles. In general I think this is a useful guide to explain the "tone/vibe" we're trying to achieve: https://ethereum.org/en/contributing/design-principles/

File: src/content/contributing/style-guide/index.md (hunk 15 above)
Comment (samajammin) [reply src/content/contributing/style-guide/index.md:109]: Agreed, I don't think capitalizing mid-sentence is necessary.

**Review by samajammin — APPROVED** [review]
Great work @minimalsm!

I've made some updates, added some content & added references to the style guide on other pages. I think it's ready to merge but feel free to review my changes & make any final edits beforehand.

**Review by wackerow — APPROVED** [review]

**Comment by minimalsm** [comment]
Bringing this in for the deploy. Will make necessary changes in a patch deploy 👍
